### PR TITLE
Project: Weather App

### PR DESCRIPTION
**Pull request by LamiaSristy**
This project is an app that will take a location and then hit the API to get all the weather information and display it.

Here I have practiced:

- Working with APIs
-  JSON
-  Async and Await 
-  JavaScript DOM manipulation.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,3 +1,89 @@
...
+    {
+      mode: 'cors',
+    },
+  );
+  if (response.status === 400) {
+    throwErrorMsg();
+  } else {
+    errorTxt.style.display = 'none';
+    const weatherData = await response.json();
+    console.log('WeatherData');
```
Comment (Forison) [inline src/index.js]: remove all `console.log` from code

**Review by Forison — CHANGES_REQUESTED** [review]
## Status::Changes required❌
# 🙋‍♂️ @LamiaSristy 
#### You have done a great job on this one so far, Your code is very clean and your UI was awesome, however, there are a few issues we have to address to get this project approved🤗.
* [x] It has a toggle control to determine if the result will be shown in Celsius or Fahrenheit. Currently, it displays in both units, let adjust it to display in one unit ie degree, and then display in Fahrenheit when clicked.
* [x] It changes its appearance based on the received data ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))
Let us make users experience different appearances based on the temperature, maybe the color of opacity could be tweak to change in color based on the temperature range, etc...
* [x] Let us remove all `console.log()` you risk exposing vital information of your application in production make sure you get rid of them after development.
**[OPTIONAL]**
* [ ] Add `"start": "webpack-dev-server --mode development --open"` your script file to enable application to be launch with the `npm start` command.
* [ ] it has a ‘loading’ component that displays from the time the form is submitted until the information comes back from the API ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/loading_component_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))
* [ ] it fetches the current geolocation of the user when the app is first accessed and fetches the weather from that location
#### That is it, kindly make the change and submit a re-review🤗.
#### Happy coding😎
Addo Forison

File: src/index.js (hunk 1 above)
Comment (LamiaSristy) [reply src/index.js]: Done.

**Review by shloch — APPROVED** [review]
## Hello @LamiaSristy 

Great job bringing all the corrections proposed by former TSE. It's all smooth now and working great
weather

You can merge this.
Happy coding

**Pull request merged**